packages/integrations/useSortable/component.ts

```diff
@@ -0,0 +1,30 @@
+import { type PropType, defineComponent, h, reactive, ref } from 'vue-demi'
+import { useVModel } from '@vueuse/core'
+import { type UseSortableOptions, useSortable } from '.'
+
+export const UseSortable = /* #__PURE__ */ defineComponent({
+  name: 'UseSortable',
+  props: {
+    modelValue: {
```

Comment:
There is an error when using it in a Vue2 project, because in Vue2 it was `value` instead of `modelValue`. The `v-model` could be replaced by a props called `list`.